src/transformers/models/convnext/modeling_tf_convnext.py

```diff
@@ -300,29 +301,16 @@ def call(
         )
         return_dict = return_dict if return_dict is not None else self.config.use_return_dict
 
-        inputs = input_processing(
-            func=self.call,
-            config=self.config,
-            input_ids=pixel_values,
-            output_hidden_states=output_hidden_states,
-            return_dict=return_dict,
-            training=training,
-            kwargs_call=kwargs,
-        )
-
-        if "input_ids" in inputs:
-            inputs["pixel_values"] = inputs.pop("input_ids")
-
```

Comment:
Was a bit unsure about the removal of the upper two lines (313,314), but according to the tests it seems safe to delete them, as they don't seem to serve anymore purpose now

Reply:
Correct, with the decorator these lines are not needed 😎